.devcontainer/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ghcr.io/commaai/openpilot-base:latest
 
 # remove gitconfig if exists, since its gonna be replaced by host one
-RUN rm -f /root/.gitconfig
+RUN rm -f /home/batman/.gitconfig
```

Comment:
this line is not needed anymore in that case, since it doesn't exist in parent image